consensus/src/sync/syncer.rs

```diff
@@ -324,7 +331,15 @@ impl<N: Network, M: MacroSync<N::PeerId>, L: LiveSync<N>> Stream for Syncer<N, M
                     debug!(%peer_id, "Macro sync returned incompatible peer");
                     self.check_incompatible_peer(peer_id);
                 }
-                None => {}
+                Some(MacroSyncReturn::Conflicting(peer_id)) => {
+                    debug!(%peer_id, "Macro sync returned conflicting peer");
+                    let mut syncing_peers = self.live_sync.peers();
+                    syncing_peers.push(peer_id);
+                    syncing_peers.append(&mut self.macro_sync.peers());
+
+                    self.macro_sync.fallback(syncing_peers);
+                }
+                _ => {}
```

Comment:
Isn't it better to keep the `None` to make the compiler alert us if we add another `enum` variant?